### ISSUE TITLE
 x = torch.cat([z,l,k],dim=1)

### ISSUE THREAD

**Issue by praveenkumarchandaliya** [pytorch/pytorch#6545]
PyTorch GitHub Issues Guidelines
--------------------------------

We like to limit our issues to bug reports and feature requests. If you have a question or would like help and support, please visit our forums: https://discuss.pytorch.org/

If you are submitting a feature request, please preface the title with [feature request].

When submitting a bug report, please include the following information (where relevant):
- PyTorch or Caffe2:
- OS:
- PyTorch version:
- How you installed PyTorch (conda, pip, source):
- Python version:
- CUDA/cuDNN version:
- GPU models and configuration:
- GCC version (if compiling from source):
- CMake version:
- Build command you used (if compiling from source):
- Versions of any other relevant libraries:

In addition, including the following information will also be very helpful for us to diagnose the problem:
- A script to reproduce the bug. Please try to provide as minimal of a test case as possible.
- Error messages and/or stack traces of the bug
- Context around what you are trying to do

**Comment by praveenkumarchandaliya** [comment]
x = torch.cat([z,l,k],dim=1)
RuntimeError:Expected objet of type Variable torch.cuda.FloatTensor
type variable torch cuda.LongTensor at position #2 for iterable argument tensors
this proble occure with pytorch 0.30

**Comment by fmassa** [comment]
The problem, as mentioned in the error message, is that one of your tensors has incompatible type (tensor number 2 is a long tensor).
Do something like
```python
x = torch.cat([z,l,k.float()],dim=1)
```

Also, those questions are better suited for our forums at https://discuss.pytorch.org/
